Sources/SwiftSyntaxBuilderGeneration/Templates/BuildableBaseProtocolsFile.swift

```diff
@@ -30,67 +30,67 @@ let buildableBaseProtocolsFile = SourceFile {
     let syntaxType = SyntaxBuildableType(syntaxKind: "Syntax")
     let isSyntax = type == syntaxType
     // Types that the `*Buildable` conforms to
-    let buildableConformances: [String] = [type.expressibleAs, type.listBuildable] + (isSyntax ? [] : [syntaxType.buildable])
-    let listConformances: [String] = isSyntax ? [] : [syntaxType.listBuildable]
+    let buildableConformances: [String] = [type.expressibleAsBaseName, type.listBuildableBaseName] + (isSyntax ? [] : [syntaxType.buildableBaseName])
+    let listConformances: [String] = isSyntax ? [] : [syntaxType.listBuildableBaseName]
```

Comment:
Previously we relied on these types being non-optional. Since optional types wouldn't make sense e.g. in conformance lists and dealing with strings is easier, we use the base names now in a few more places (this shouldn't change anything in the generated code).